Presentation.Web/app/components/org/structure/org-unit-migration.component.ts

```diff
@@ -0,0 +1,431 @@
+        relevantSystemTableConfig: IMigrationTableColumn[];
+        responsibleSystemTableConfig: IMigrationTableColumn[];
+
+        private isBusy = false;
+
+        static $inject: string[] = ["organizationRegistrationsService", "organizationApiService", "notify"];
+        constructor(private readonly organizationRegistrationsService: Services.Organization.IOrganizationRegistrationsService,
+            private readonly organizationApiService: Services.IOrganizationApiService,
+            private readonly notify) {
+        }
+
+        $onInit() {
+            if (this.organizationId === null) {
+                console.error("missing attribute: 'organizationId'");
+            }
+            if (this.unitId === null) {
+                console.error("missing attribute: 'unitId'");
+            }
+            if (this.unitName === null) {
+                console.error("missing attribute: 'unitName'");
+            }
+
+            this.createTableConfigurations();
+            this.setupOptions();
+            this.getData();
+
+            this.orgUnits = [];
+            this.organizationApiService.getOrganizationUnit(this.organizationId).then(result => {
+                this.orgUnits = this.orgUnits.concat(Helpers.Select2OptionsFormatHelper.addIndentationToUnitChildren(result, 0));
+            });
+        }
+
+        deleteSelected() {
+            if (this.isBusy) {
+                return;
+            }
+            if (!confirm('Er du sikker på, at du vil slette de valgte registreringer?')) {
+                return;
+            }
+            this.isBusy = true;
+
+            const request = this.createChangeRequest();
+            this.organizationRegistrationsService.deleteSelectedRegistrations(this.organizationId, this.unitId, request)
+                .then(() => {
+                    this.refreshData(); 
+                },
+                error => {
+                    console.error(error);
+                    this.notify.addErrorMessage("Delete selected failed");
+                });
+
+            this.isBusy = false;
+        }
+
+        transfer() {
+            if (this.isBusy) {
+                return;
+            }
+            if (!this.selectedOrg?.id)
+                return;
+            if (!confirm(`Er du sikker på, at du vil overføre de valgte registreringer til "${this.selectedOrg.text}"?`)) {
+                return;
+            }
+            this.isBusy = true;
+
+            const request = this.createChangeRequest();
+            this.organizationRegistrationsService.transferSelectedRegistrations(this.organizationId, this.unitId, this.selectedOrg.id, request)
+                .then(() => {
+                    this.selectedOrg = null;
+                    this.refreshData();
+                }, error => {
+                    console.log(error);
+                    this.notify.addErrorMessage("Transfer failed");
+                });
+
+            this.isBusy = false;
```

Comment:
You are setting it too early... remember.. your service methods are async so setting this state must be postponed until after the resolution of the promises... same for your other mutation calls